functions/deleteShow.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+const AWS = require('aws-sdk');
+const httpResponse = require('../httpResponse')
+
+
+module.exports.handle = async (event) => {
```

Comment:
Não precisa manter o nome padrão pode trocar sem problemas.

```suggestion
module.exports.updateEvent = async (event) => {
```